bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/ObjectType.java

```diff
@@ -43,4 +45,11 @@
      */
     boolean isIsolated(String methodName);
 
+    /**
+     * Provides copy of type ids of the object.
+     *
+     * @return copy of type id set.
+     */
+    BTypeIdSet getTypeIdSet();
```

Comment:
Should this return `TypeIdSet` instead of `BTypeIdSet` since it's an internal one?